interaction/src/test/java/io/honeycomb/opentelemetry/android/interaction/ExampleUnitTest.kt

```diff
@@ -0,0 +1,16 @@
+package io.honeycomb.opentelemetry.android.interaction
+
+import org.junit.Assert.*
+import org.junit.Test
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, 2 + 2)
+    }
+}
```

Comment:
As with #24, it's kinda silly to have this here, but it's less annoying than adding it later.